plugins/minigames.json

```diff
@@ -967,6 +967,139 @@
           "md5sum": "1cbe5b3e85b5dfcee1eb322f33568fd4"
         }
       }
+    },
+    "Avalanche": {
```

Comment:
let's try stick to snake case plugin names here and for rest of the plugins in this PR

Reply:
AAh ok